src/common/transformations/src/transformations/common_optimizations/moc_transformations.cpp

```diff
@@ -205,6 +206,7 @@ bool ov::pass::MOCTransformations::run_on_model(const std::shared_ptr<ngraph::Fu
     ADD_MATCHER(common_fusions, DepthToSpaceFusion)
     ADD_MATCHER(common_fusions, ShuffleChannelsFusion, !m_use_shapes)
     ADD_MATCHER(common_fusions, NonZeroHorizontalFusion)
+    ADD_MATCHER(common_fusions, AdaptivePoolToReduce)
```

Comment:
I would recommend to add one integration test case for sub-graph with Adaptive pooling to ensure that IR will contain Reduce. It will prevent performance degradation in the future for this case.

Reply:
Also, is NNCF tool fine with this change and about other plugins? May be we need to switch this transformation precisely only for CPU?